_posts/2017/2017-16-01-testing-async-interactions.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: post
+title: "Testing asynchronous interactions in distributed systems"
```

Comment:
I tried hard, this is my top-scorer... 
(77) "Writing better messaging verifications: A simple how-to"